plugin/clone/src/clone_client.cc

```diff
@@ -26,6 +26,7 @@ Clone Plugin: Client implementation
 
 */
 #include <inttypes.h>
+#include <cassert>
```

Comment:
Suspicious formatting (clang-format sorts headers alphabetically in each section)

Reply:
it is weird. I used internal standard clang-format tooling for formatting.

Reply:
OK then